ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -24,113 +24,71 @@
+    />
+    <Li.menu>
+      <PopupMenu @name="engine-menu">
+          <Confirm as |c|>
+          <nav class="menu">
+            <ul class="menu-list">
+              <li class="action">
+                <LinkTo @route="vault.cluster.secrets.backend.configuration" @model={{backend.id}}>
+                  View configuration
+                </LinkTo>
+                </li>
+                {{#unless (eq backend.type "cubbyhole")}}
+                <li class="action">
+                  <c.Message
+                    @id={{backend.id}}
+                    @triggerText="Disable"
+                    @message="Any data in this engine will be permanently deleted."
+                    @title="Disable engine?"
+                    @confirmButtonText="Disable"
+                    @onConfirm={{perform disableEngine backend}}
+                    data-test-engine-disable="true"
+                    />
+                </li>
+                {{/unless}}
+              {{#if item.updatePath.isPending}}
+                <li class="action">
+                  <button disabled type="button" class="link button is-loading is-transparent">
+                      loading
+                    </button>
+                  </li>
+              {{/if}}
+              </ul>
+            </nav>
+          </Confirm>
+      </PopupMenu>
+    </Li.menu>
+  </LinkableItem>
   {{/let}}
 {{/each}}
+
 {{#each unsupportedBackends as |backend|}}
-  <div class="list-item-row" data-test-secret-backend-row={{backend.id}}>
-    <div class="level is-mobile">
-      <div class="level-left">
-        <div>
-          <div data-test-secret-path class="has-text-weight-semibold has-text-grey">
-            <ToolTip @horizontalPosition="left" as |T|>
-              <T.trigger>
-                <Icon
-                  @glyph={{or backend.engineType "secrets"}}
-                  class="has-text-grey-light"
-                />
-              </T.trigger>
-              <T.content @class="tool-tip">
-                <div class="box">
-                  {{backend.engineType}}
-                </div>
-              </T.content>
-            </ToolTip>
-            {{backend.path}}
-          </div>
-          <code class="has-text-grey is-size-8">
-            {{backend.accessor}}
-          </code>
-        </div>
-      </div>
-      <div class="level-right is-flex is-paddingless is-marginless">
-        <div class="level-item">
-          <PopupMenu name="engine-menu">
+  <LinkableItem 
+    data-test-secret-backend-row={{backend.id}}
+    @disabled={{true}}
+  as |Li|>
+    <Li.content
+      @accessor={{if (eq backend.options.version 2) (concat "v2 " backend.accessor) backend.accessor}}
+      @description={{backend.description}}
+      @glyphText={{backend.engineType}}
+      @glyph={{or (if (eq backend.engineType "kmip") "secrets" backend.engineType) "secrets"}}
+      @title={{backend.path}} 
+      />
+    <Li.menu>
+      <PopupMenu name="engine-menu">
```

Comment:
The indentation here seems off?  It looks like more than one indentation for <Confirm ...